compiler-cli/src/new/tests.rs

```diff
@@ -297,3 +297,31 @@ fn conflict_with_existing_files() {
         })
     );
 }
+
+#[test]
+fn skip_existing_git_files_when_skip_git_is_true() {
+    let tmp = tempfile::tempdir().unwrap();
+    let path = Utf8PathBuf::from_path_buf(tmp.path().join("my_project")).expect("Non Utf8 Path");
+
+    crate::fs::mkdir(&path).unwrap();
+    let file_path = PathBuf::from(&path).join(".gitignore");
+
+    let _ = std::fs::File::create(file_path);
```

Comment:
Use `.unwrap();` rather than discarding the result as otherwise this operation could fail and we wouldn't know. 🙏

Reply:
Updated checking that the result is ok